yare-model-validators/src/test/java/com/sabre/oss/yare/model/validator/ReferenceValidatorTest.java

```diff
@@ -65,6 +65,20 @@ void shouldFailOnMissingPredicate() {
         );
     }
 
+    @Test
+    void shouldFailOnNullPredicate() {
```

Comment:
This is duplication of existing shouldFailOnMissingPredicate test. Please remove it.